lib/node_modules/@stdlib/math/base/special/factorial2/test/test.js

```diff
@@ -126,7 +127,7 @@ tape( 'the function evaluates the factorial function (decimal values)', function
 		v = factorial2( x[ i ] );
 		delta = abs( v - expected[ i ] );
 		tol = 2.0e-14 * Math.max( 1.0, abs( v ), abs( expected[ i ] ) );
```

Comment:
This line should be

```
tol = <num> * EPS * abs( expected[ i ] );
```
For example,
```
tol = 100.0 * EPS * abs( expected[ i ] );
```
The reason why we scale epsilon is that doing so provides more context as to how much results are off by and lessens the presence of so-called "magic" numbers. For instance, if we know that
```
actual = <number>;
expected = 1.0;
tol = 1.0 * EPS * abs( expected );
delta = abs( actual - expected );
bool = ( delta <= tol );
```
we know that actual value differs from the expected by only `1` bit. Similarly for other values, although the correspondence between units epsilon and number of differing bits is not 1:1.